src/component.ts

```diff
@@ -64,5 +64,10 @@ export function componentFactory (
   const Super = superProto instanceof Vue
     ? superProto.constructor as VueClass
     : Vue
-  return Super.extend(options)
+  const rv = Super.extend(options);
+
+  for(let sttc in Component)
+    if(!(sttc in function() {}) && Component.hasOwnProperty(sttc))
+      rv[sttc] = Component[sttc];
```

Comment:
Can we rename `rv` to `Extended` and also `sttc` to `key`? I cannot see what the current name means...